command/git/commands.go

```diff
@@ -144,3 +144,19 @@ func (g *Git) Status(opts ...string) *command.Model {
 func (g *Git) Config(key string, value string) *command.Model {
 	return g.command("config", key, value)
 }
+
+// SparseCheckoutInit initializes the sparse-checkout config file.
+func (g *Git) SparseCheckoutInit(cone bool) *command.Model {
```

Comment:
Why not the usual `opts ...string` Is this `--cone` special?

Reply:
I have seen both ways used here and since `--cone` is the only flag for [init](https://git-scm.com/docs/git-sparse-checkout#Documentation/git-sparse-checkout.txt-eminitem) I've went with this option.